.github/workflows/ci-rust.yml

```
@@ -146,13 +147,30 @@ jobs:
         run: echo OPENSSL_SRC_PERL=C:/Strawberry/perl/bin/perl >> $GITHUB_ENV
         timeout-minutes: 1
 
+      - name: Install winfsp
+        if: startsWith(matrix.os, 'windows')
+        shell: bash
+        run: |
+          set -eux
+          choco install winfsp -y --version=${{ env.WINFSP_VERSION }}
+          mkdir -vp ${{ runner.temp }}
+          curl -L https://github.com/winfsp/winfsp/releases/download/v2.0/winfsp-tests-${{ env.WINFSP_VERSION }}.zip -o ${{ runner.temp }}/winfsp-tests.zip
+          unzip ${{ runner.temp }}/winfsp-tests.zip -d ${{ runner.temp }}/winfsp-tests.exe
```

`-d` option is to specify an output directory here, `winfsp-test.exe` will be a directory (not a file ;) )

```suggestion
          unzip ${{ runner.temp }}/winfsp-tests.zip -d ${{ runner.temp }}/winfsp-tests
```

---

Ty